src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -224,18 +224,27 @@ export const restoreBrowserTrackStates: ActionCreator<ThunkAction<
 
 export const openBrowserNav = createAction(
   'browser/open-browser-navigation',
-  () => {
+  (activeGenomeId: string) => {
     analyticsTracking.trackEvent({
       category: 'browser_navigation',
       label: 'open_browser_navigation',
       action: 'clicked'
     });
+
+    return {
+      activeGenomeId
+    };
   }
-)();
+)<{ activeGenomeId: string }>();
 
 export const closeBrowserNav = createAction(
-  'browser/close-browser-navigation'
-)();
+  'browser/close-browser-navigation',
+  (activeGenomeId: string) => {
+    return {
+      activeGenomeId
+    };
+  }
+)<{ activeGenomeId: string }>();
```

Comment:
what's the purpose of this change?

Reply:
I don't that there is any good purpose behind this change. I have reverted it.